samples/templates/default-azuredeploy.json

```diff
@@ -126,14 +126,22 @@
         },
         "fhirVersion": {
             "type": "string",
-            "defaultValue": "Stu3",
+            "defaultValue": "R4",
             "allowedValues": [
                 "Stu3",
                 "R4"
             ],
             "metadata": {
                 "description": "Only applies when specifying Repository Url"
             }
+        },
+        "storageAccountName": {
```

Comment:
Should we derive the storage account name instead of asking user to enter it so it behaves like other dependencies (e.g., KeyVault, AppInsights and etc)?

I added the storage account integration for the dicom-server template: https://github.com/microsoft/dicom-server/pull/23/files

Reply:
Update the template to generate storage account name. Also added an option for users to choose whether they want to enable export.